src/components/tables/AccessGrants/AccessLinks/Dialog/GenerateInvitation.tsx

```diff
@@ -91,82 +163,218 @@ function GenerateInvitation({
+                        gap: 2,
+                        minWidth: 180,
                     }}
-                    required
-                    validateOnLoad
-                />
-            </Grid>
+                >
+                    <Box sx={{ pt: 1 }}>
+                        <PrefixedName
+                            prefixOnly
+                            defaultPrefix
+                            disabled={objectRoles?.length === 1}
+                            label={intl.formatMessage({
+                                id: 'admin.users.prefixInvitation.label.tenant',
+                            })}
+                            onChange={onChange}
+                        />
+                    </Box>
 
-            <Grid item xs={4} md={2}>
-                <AutocompletedField
-                    label={intl.formatMessage({
-                        id: 'admin.users.prefixInvitation.label.capability',
-                    })}
-                    required
-                    options={capabilityOptions}
-                    defaultValue={capabilityOptions[0]}
-                    changeHandler={handlers.setGrantCapability}
-                />
-            </Grid>
+                    <AutocompletedField
+                        label={intl.formatMessage({
+                            id: 'admin.users.prefixInvitation.label.capability',
+                        })}
+                        required
+                        options={capabilityOptions}
+                        defaultValue={capabilityOptions[0]}
+                        changeHandler={handlers.setGrantCapability}
+                    />
+                </Box>
+
+                <Divider flexItem orientation="vertical" />
 
-            <Grid item xs={4} md={2}>
-                <AutocompletedField
+                <Box sx={{ flex: 1 }}>
+                    <RadioGroup
+                        sx={{
+                            pt: 1,
+                            gap: 1.5,
+                        }}
+                        value={accessScope}
+                        onChange={(event) => setAccessScope(event.target.value)}
+                    >
+                        <RadioOption
+                            value="full"
+                            isSelected={accessScope === 'full'}
+                            label={
+                                <span>
+                                    {intl.formatMessage({
+                                        id: 'admin.users.prefixInvitation.label.scope.full',
+                                    })}
+                                    <span
+                                        style={{
+                                            fontFamily: 'monospace',
+                                            fontSize: 12,
+                                        }}
+                                    >
+                                        {clampedPrefix}
+                                    </span>
```

Comment:
We have `TechnicalEmphasis` which _might_ be close to what you are wanting here. So I'd see if that can do what you're hoping.